icons/autodesk.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Autodesk icon</title><path d="m14.574 1.0203c-0.097-0.096997-0.29099-0.58198-0.97097-0.58198h-6.7038s0.97097 0.096997 1.36 1.068c0 0 1.069 2.5269 2.0399 4.9558 2.8179 6.6068 7.1898 17.099 7.1898 17.099h6.5108c0.097-0.097-9.3267-22.443-9.4247-22.54zm-8.8407 0.87497-5.3438 12.631c-0.29199 0.87497-0.097997 1.9439 1.457 1.9439h4.1779l3.6919-8.8417c-1.166-2.9149-2.1359-5.2478-2.1359-5.2478-0.096997-0.29199-0.38899-1.069-0.97197-1.069-0.58298 0-0.77698 0.48598-0.87397 0.58298zm-0.097997 15.643h-4.4689c-0.77698 0-1.166-0.48598-1.166-0.48598 0.77698 1.36 3.0119 5.6358 3.0119 5.6358 0.38899 0.48598 0.77698 0.77698 1.36 0.77698 1.263 0 3.2069-1.263 3.2069-1.263l7.4808-4.6639z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Autodesk icon</title><path d="M14.5736 1.0206c-.097-.097-.291-.582-.971-.582H6.8988s.971.097 1.36 1.068c0 0 1.069 2.5269 2.0399 4.9558 2.8179 6.6068 7.1898 17.099 7.1898 17.099h6.5108c.097-.097-9.3267-22.443-9.4247-22.54zm-8.8407.875L.389 14.5266c-.292.875-.098 1.9439 1.457 1.9439H6.024l3.6919-8.8418C8.5499 4.714 7.5799 2.381 7.5799 2.381c-.097-.292-.389-1.069-.972-1.069-.5829 0-.7769.486-.8739.583zm-.098 15.643h-4.469c-.777 0-1.166-.486-1.166-.486.777 1.36 3.012 5.6358 3.012 5.6358.389.486.777.777 1.36.777 1.263 0 3.2069-1.263 3.2069-1.263l7.4808-4.664z"/></svg>
```

Comment:
Rebuilding this from source, I've got an overlap between the sections (see below). Would we be better to keep those in, or should we stick with the smoother variation of the logo @simple-icons/maintainers?

![image](https://user-images.githubusercontent.com/1830867/103171231-5ff70e00-4842-11eb-98e6-705076c34230.png)

Reply:
> Rebuilding this from source, I've got an overlap between the sections (see below). Would we be better to keep those in, or should we stick with the smoother variation of the logo?

I'm not sure what do you mean. For me, giving big differences that I'm seeing with the source, maybe this icon is outdated. Could you rebuild from source @fbernhart?

Reply:
I guess it actually wasn't outdated. The SVG we've used is [this one](https://damassets.autodesk.net/content/dam/autodesk/www/homepage/fy18/adsk-white.svg) from the (more or less) footer of [their website](https://www.autodesk.com/#). It's the monochrome version, which as well can be seen used here: https://brand.autodesk.com/brand-elements/autodesk-logo#acceptable-uses

The monochrome version is slightly different to the colored one as it has gaps.

![image](https://user-images.githubusercontent.com/70264417/103303359-10f9d600-4a06-11eb-8646-93f74ffbeb6a.png)


Anyway, I've rebuilt the icon now with the monochrome version as source. @adamrusted please have a look at it. 😊